README.md

```diff
@@ -1,86 +1,157 @@
-- Be actively kind.
-- We're here to help: Andy Sparks (@andy) and Josh Levy (@jlevy)
+- The main goal is to be **helpful**.
+- But also, be kind.
+- We’re here to help.
+  Your administrators are Andy Sparks (@andy) and Josh Levy (@jlevy).
 - Abusive behavior is never tolerated.
-- Data published to the Holloway Community is hosted at the discretion of the administrators, and may be removed.
-- Violations of this code may result in permanent expulsion from the Holloway Community Slack.
+- Data published to the Holloway Community is hosted at the discretion of the
+  administrators, and may be removed.
+- Violations of this code may result in permanent expulsion from the Holloway Community
+  Slack.
 
 ## Our Standards
 
 Examples of behavior that contribute to a helpful, positive environment include:
 
-- Using welcoming and inclusive language
-- Being respectful of differing viewpoints and experiences
-- Gracefully accepting constructive criticism
-- Focusing on what is best for the community
-- Showing empathy towards other community members
+- Using welcoming and inclusive language.
+- Being respectful of differing viewpoints and experiences.
+- Gracefully accepting constructive criticism.
+- Focusing on what is best for the community.
+- Showing empathy toward other community members.
 
 Examples of unacceptable behavior by participants include:
 
-- The use of sexualized language or imagery and unwelcome sexual attention or advances
-- Trolling, insulting or derogatory comments, and personal or political attacks
-- Public or private harassment
-- Publishing others' private information, such as physical or electronic address, without explicit permission
-- Other conduct which could reasonably be considered inappropriate in a professional setting
+- The use of sexualized language or imagery and unwelcome sexual attention or advances.
+- Trolling, insulting or derogatory comments, and personal or political attacks.
+- Public or private harassment.
+- Publishing others' private information, such as a physical or electronic address, without
+  explicit permission.
+- Other conduct which could reasonably be considered inappropriate in a professional
+  setting.
 
 ## Scope
 
-We expect all members of the Holloway Community, including administrators, users, and contributors to abide by this Code of Conduct at all times in the Holloway Community Slack Group. 
+We expect all members of the Holloway Community, including administrators, users, and
+contributors, to abide by this Code of Conduct at all times in the Holloway Community
+Slack Group.
 
-The definitions of various subjective terms such as "discriminatory," "hateful," "or "confusing," will be decided at the sole discretion of the Holloway Community administrators: Andy Sparks and Josh Levy.
+The definitions of various subjective terms such as “discriminatory,” “hateful,” “or
+"confusing,” will be decided at the sole discretion of the Holloway Community
```

Comment:
None of those words are used elsewhere in the CoC. Are they supposed to appear, or should the words be "harassment," "trolling," "inappropriate," or other?